README.md

```diff
@@ -27,6 +27,10 @@ From Administrator's point of view, every Refund request results in creating two
 * Credit Memo - a document representing a list of refunded items (downloadable and sent to Customer via .pdf file)
 * Refund Payment - entity representing payment in favour of the Customer
 
+## Pre - requirements
+
+In order to be able to generate PDF files of refunds, [wkhtmltopdf](https://wkhtmltopdf.org/) must be installed.
```

Comment:
Maybe we can mention that it's used by `KnpSnappyBundle`, not by plugin directly? And therefore it needs to be configured as mentioned in this bundle installation instruction https://github.com/KnpLabs/KnpSnappyBundle#configuration